src/Service.Tests/SqlTests/SqlTestBase.cs

```diff
@@ -101,6 +101,13 @@ public abstract class SqlTestBase
                 _ => TestHelper.AddMissingEntitiesToConfig(runtimeConfig, "magazine", "foo.magazines"),
             };
 
+            // Add table name collision testing entity to the config
+            runtimeConfig = DatabaseEngine switch
+            {
+                TestCategory.MYSQL => runtimeConfig,
+                _ => TestHelper.AddMissingEntitiesToConfig(runtimeConfig, "bar_magazine", "bar.magazines", new string[] { "upc" })
```

Comment:
need named parameters , assuming `new string[] { "upc" }` represents a column name?

Any reason not to add this to the test config files directly and, if necessary, make it a duplicate of an entity in the test config file(s)? I understand magazine is already added as a "missing entity", but we may not need to continue that pattern as it maintains entities in two separate places.

Reply:
This is because MySql handles schemas differently than the rest of the the DB types that we have. We can not have the kinds of different schemas in our config file that is included for MySql. Instead we include a config file that represents the intersection between what is valid for all DB types, and then modify it here. This avoids having to maintain multiple config files for testing.